nemo_rl/distributed/ray_compiled_graph.py

```
@@ -0,0 +1,837 @@
+            try:
+                executor = CompiledGraphExecutor(
+                    workers=self.worker_group._workers,
+                    sharding_annotations=self.worker_group.sharding_annotations,
+                    method_name=method_name,
+                    overlap_communication=self.config["overlap_communication"],
+                )
+                self.compiled_executors[method_name] = executor
+                logger.info(
+                    f"✅ Created compiled graph executor for method '{method_name}'"
+                )
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)